### DISCUSSION TITLE
Appimage doesn't respect portable.ini

### DISCUSSION THREAD

**Discussion by ghost** [PCSX2/pcsx2#7448] [General]
Is this intended?

I want everything in one folder not separated but making portable.ini doesn't seem to be respected by pcsx2 once it moved to appimage.

**Comment by stenzek** [comment]
Yes, because the executable runs out of a squashfs mounted in /tmp. If you extracted the appimage and ran it from there, it'll probably work, but then you won't have the automatic updater.

Portable mode is more of a Windows thing.

**Reply by ghost** [reply]
Ah, thanks for the information.
I didn't know that's how appimages work since every emulator moved to support it.

**Reply by stenzek** [reply]
It would be possible to implement, you'd have to check the `APPIMAGE` environment variable, get the directory, and also change the portable mode logic to target there instead of the exe directory.

**Reply by ghost** [reply]
Learnt something new today.
Got options to make it portable again.
Thanks so much.

**Comment by qurious-pixel** [comment]
@ghost573 If you create a new empty folder and copy/paste the pcsx2 AppImage, this script will create the folders need to keep all of the files in this folder. 

create ~/PCSX2 and copy pcsx2.AppImage file to this folder.

```
PCSX2DIR=$(ls *.AppImage)
mkdir ./$PCSX2DIR.{home,config}
chmod +x $PCSX2DIR
./$PCSX2DIR
```
